bot/cogs/bot.py

```diff
@@ -60,6 +62,18 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        """
+        Auto codeblock python code
+        """
+        if msg.content.count("\n") >= 3:  # more than three lines
+            try:
+                tree = ast.parse(msg.content)  # no syntax errors
+                if not all(isinstance(node, ast.Expr) for node in tree.body):  # we dont want hi\nthere\nguys\nD
+                    await self.bot.send_message(msg.author, "```Python\n{}\n```".format(msg.content))
```

Comment:
Should technically be a lower case `p`, as well as this we should probably either delete the message, or, have it send the current text in the codeblock tag.

Reply:
ok

Reply:
Regarding previous comment, the lowercase `p` was referring to the syntax block.
```py
"```python\n..."
```

Reply:
We use f-strings all over the bot, this should also be an f-string.